### PR TITLE
semih week-2

### PR DESCRIPTION
**Pull request by sigma21**
sorry for the delay @Meno-101 . hope you'll like it.

### CODE REVIEW

File: homework-submission/semih-week2/form.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        event.preventDefault();
+        const markasdoneIndex = this.refs.markasdone.value-1
+        const todos = this.state.todos;
+        todos[markasdoneIndex].done = true;
+        this.setState({todos:todos});
+    }
+        
+    render(){
+        const todos = this.state.todos;
+        return (
```
Comment (Meno-101) [inline homework-submission/semih-week2/form.js]: Perhaps some splitting the content will make this `jsx` content easier to be read.

File: homework-submission/semih-week2/form.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            <div>
+            <ol>
+            {todos.map((item,index)=>
+                {return <li>
+                <pre id={index} ref={index} style={{textDecoration: item.done ? 'line-through' : 'none'}} 
+                onClick={()=>{}}>{item.description}</pre></li>})}
```
Comment (Meno-101) [inline homework-submission/semih-week2/form.js]: 🤔
```suggestion
                >{item.description}</pre></li>})}
```

File: homework-submission/semih-week2/form.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            </ol>
+
+            <button onClick={()=> {this.createToDo()}}>Add Demo Todo</button>
+            <button onClick={()=> {this.removeToDo()}}>Remove Last Item</button>
+            <p>----------------------------------------------</p>
+            <form onSubmit={this.submitFormHandler}>
```
Comment (Meno-101) [inline homework-submission/semih-week2/form.js]: I see two forms each is responsible for a one button, and I saw in your case you have used refs, and that probably will make it easier to use an `onClick`, will make it a lot shorter , what do you think about it?

**Comment by wouterkleijn** [comment]
Thanks @Meno-101 Please address the changes @sigma21

**Comment by wouterkleijn** [comment]
Please make the necessary changes @sigma21

**Pull request closed without merging**